Update xero-bank-feeds-push-bank-transactions.md (#1412)
codatio/codat-docs: docs/integrations/bank-feeds/xero-bank-feeds/xero-bank-feeds-push-bank-transactions.md

When writing bank transactions to Xero:

- You can only write bank transactions to one target account at a time.
- Transactions must be written in chronological order.

> - Transactions can't be older than the most recent transaction available on the destination bank account.

Reviewer: Line 66 is still valid though no?

Author: Checked w/ Payments team and apparently not for Xero, (this line contradicts the section immediately below regarding historic transactions, hence the PR!)

> - Within the same request, transactions must be written in chronological order.

Reviewer: ```suggestion
- Within a single request, the array of transactions _must_ be in chronological order.
```
